### ISSUE TITLE
"LoaderOption" not callable when using joinedload()

### ISSUE THREAD

**Issue by TilmanK** [sqlalchemy/sqlalchemy2-stubs#64]
**Describe the bug**
When using joinedload the way it is described in the current documentation I get the following error:
```"LoaderOption" not callable```

**Expected behavior**
No error.

**To Reproduce**

Import joinedload as folows:
```py
from sqlalchemy.orm import joinedload
```

And then use it in a query:
```py
users = session.query(User).options(joinedload(User.address)).all()
```

**Versions.**
 - OS:
 - Python: 3.8.6
 - SQLAlchemy: 1.4.9
 - Database:
 - DBAPI:

**Have a nice day!**
Again: you too :)

**Comment by bryanforbes** [comment]
This was fixed by #43

**Comment by TilmanK** [comment]
Thanks! 😃
